lib/battle/CBattleInfoCallback.cpp

```diff
@@ -855,6 +855,24 @@ AccessibilityInfo CBattleInfoCallback::getAccesibility() const
 		ret[BattleHex(0, y)] = EAccessibility::SIDE_COLUMN;
 	}
 
+	//special battlefields with logically unavailable tiles
+	std::vector<BattleHex> impassableHexes;
+	if(battleGetBattlefieldType().num == BFieldType::SHIP_TO_SHIP)
+		impassableHexes = 
+		{ 

```

Comment:
```
if(a)
	if(b)
		if(c)
			doSth();
```
->
```
if(a)
{
	if(b)
		if(c)
			doSth();
}
```
If whole (inner) contraption is more than 1 line then outer brackets required